utilities/roswtf/test/check_roswtf_command_line_online.py.orig

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
```

Comment:
I think this file `...py.orig` might have been accidentally committed. Please remove it.